2.evaluate_data/identify_sc_outlier_per_plate/identify_per_plate_anomalies.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
```

Comment:
Consider the following:
1. a one line comment for what this script does
2. A conda env activation statement. This will make sure the script is run in the correct env
3. A small readme for this script, especially if you plan to pipe the std out to a log-like file

Reply:
Just added 1 and 3. Number 2 would throw an error, even after specifying the shell.